### ISSUE TITLE
Randomizer

### ISSUE THREAD

**Issue by tkram01** [lemire/externalsortinginjava#30]
Could this be modified to randomize huge files instead of sort them much like "sort -R" does?

**Comment by lemire** [comment]
Yes. The sort of the blocks should become a shuffle and then we need some slightly tricky merge process (not hard but it needs to be done well).

If you are interested, I can help with the algorithm.

It is probably about 20, 30 lines of code in the end.

**Comment by tkram01** [comment]
Yes - I'm happy to work on it with some guidance on the merge process. Thanks!

**Comment by lemire** [comment]
@tkram01 Great. So I suggest you start by forking the project. Create a new class that looks like the existing sorted, give it a good name. Then ping me (just write  @lemire in an issue within your fork).
